styles.js

```diff
@@ -1,14 +0,0 @@
-import { createGlobalStyle } from "styled-components";
-
-export default createGlobalStyle`
-  *,
-  *::before,
```

Comment:
Removing `*::before` and `*::after` might lead to some unexpected behaviour - just as a remark